src/commons/utils/Constants.ts

```diff
@@ -39,7 +39,10 @@ const sicpBackendUrl =
 const workspaceSettingsLocalStorageKey = 'workspace-settings';
 
 // For achievements feature (CA - Continual Assessment)
-const caFulfillmentLevel = process.env.REACT_APP_CA_FULFILLMENT_LEVEL || 0;
+// TODO: remove dependency of the ca levels on the env file
```

Comment:
i think TODO can be removed now since now test env no longer needs to specify ca level in env file?